controllers/admin/AdminModulesController.php

```diff
@@ -124,10 +124,10 @@ public function __construct()
     public function checkCategoriesNames($a, $b)
```

Comment:
Aside, `check` verb and plurals for comparator function is inusual right?
```suggestion
    public function checkCategoriesNames($a, $b) {
        // ADD Deprecated notice!
        return $this->compareCategoryName($a, $b);
    }
    public function compareCategoryName($a, $b)
```

Reply:
but the old legacy controller is not used any more right? So we can just close this issue and PR?

Reply:
Hi @Prestaworks, for now not all legacy controllers are migrated to symfony controllers 😔
I have one dream... complete this long long migration, but it's a lot of work!

Reply:
@boherm yes, however :) "checkCategoriesNames" is only used in the constructor of the AdminModulesController.php, removing the code we are talking about

`    // public function checkCategoriesNames($a, $b)
    // {
    //     if ($a['name'] === $this->trans('Other Modules')) {
    //         return true;
    //     }

    //     return (bool) ($a['name'] > $b['name']);
    // }`
the module manager controller is still working, also the entire construct function can be removed from the file and the page in BO is still working (PS 8.1.0).

it's used to sort the categories in list_modules_categories that seems to be used in /themes/default/template/controolers/modules/page.tpl,, however, this tpl file is not used either as the controller for module manager uses twig now. So I think we can close this PR and issue as it's fixed by changing to symfony.

Reply:
My Suggested change is to rename `checkCategoriesNames` to `compareCategoryName`.
Since `checkCategoriesNames` is public, some no official module could depends on it, then we have to keep the old function as deprecated.
The new comparator, if possible, should be private.

Reply:
I deleted the file AdminModulesController.php, cleared cache. the module manager page still works fine, I am able to install modules also. I don't think the file is used any more?

Reply:
Then I think it should be deleted from codebase

Reply:
@davidglezz Yes, I agree with you, the file and connected tpl files should be deleted.